projects/packages/jetpack-mu-wpcom/.phan/baseline.php

```diff
@@ -9,29 +9,31 @@
  */
 return [
     // # Issue statistics:
-    // PhanUndeclaredFunction : 75+ occurrences
+    // PhanUndeclaredFunction : 100+ occurrences
```

Comment:
What is phan baseline? Is this stuff we should be changing? It's in another package? :-|

Reply:
This is auto-generated by `phan`, which I am not having "fun" with atm. Yes, that was a "pun".

**phan** was added for static analysis in mono-repo recently.